### PR TITLE
Feature: Move players to company accepting take-over offer.

### PR DESCRIPTION
**Pull request by SamuXarick**
https://www.tt-forums.net/viewtopic.php?p=1201341#p1201341

Adds a company setting which allows to move the players of the offering company to the purchasing company when the company accepts the take-over offer.

When loading savegames before this patch, set it to disabled for each company.
Defaulted to disabled.

### CODE REVIEW

**Comment by TrueBrain** [comment]
I see two issues with this PR, from a functional perspective:
- If we would be to do this, I would expect I can say this during liquidation. Having it a generic setting would be surprising in many cases. After all, the players controlling the company failed to make a profitable company. Chances are slim they will be an asset to your company
- Gameplay-wise, having this ability is weird. The company went bankrupt for a reason. Why would you also take over the people controlling the company. And why is the serving dictating this.

In other words, we should not be implementing this solution. Things that might be worth exploring, as alternative solution:

- Bankrupt window showing: yes / yes-with-players / no
- Possibility to invite spectators to join your company

Anyway, tnx for the contribution; sadly, this is not a solution we are going with.
